test/cql-pytest/test_describe.py

```diff
@@ -0,0 +1,344 @@
+                FROM system_schema.keyspaces
+                WHERE keyspace_name='{new_ks}'
+            """).one()
+            assert ks_row == new_ks_row
+
+            ks_scylla_row = cql.execute(f"""
+                SELECT storage_options, storage_type
+                FROM system_schema.scylla_keyspaces
+                WHERE keyspace_name='{ks}'
+            """).one()
+            new_ks_scylla_row = cql.execute(f"""
+                SELECT storage_options, storage_type
+                FROM system_schema.scylla_keyspaces
+                WHERE keyspace_name='{new_ks}'
+            """).one()
+            assert ks_scylla_row == new_ks_scylla_row
+        finally:
+            cql.execute(f"DROP KEYSPACE {new_ks}")
+
+# Test that `DESC TABLE {tbl}` contains appropriate create statement for table
+def test_desc_table(cql, test_keyspace, randomizer_seed):
+    with new_random_table(cql, test_keyspace) as tbl:
+        desc = cql.execute(f"DESC TABLE {tbl}")
+        desc_stmt = desc.one().create_statement
+
+        new_tbl = f"{test_keyspace}.{unique_name()}"
+        new_desc_stmt = desc_stmt.replace(tbl, new_tbl)
+
+        try:
+            cql.execute(new_desc_stmt)
+            tbl_row = cql.execute(f"""
+                SELECT keyspace_name, bloom_filter_fp_chance, caching,
+                comment, compaction, compression, crc_check_chance,
+                dclocal_read_repair_chance, default_time_to_live, extensions,
+                flags, gc_grace_seconds, max_index_interval,
+                memtable_flush_period_in_ms, min_index_interval, read_repair_chance,
+                speculative_retry
+                FROM system_schema.tables
+                WHERE keyspace_name='{test_keyspace}' AND table_name='{get_name(tbl)}'
+            """).one()
+            new_tbl_row = cql.execute(f"""
+                SELECT keyspace_name, bloom_filter_fp_chance, caching,
+                comment, compaction, compression, crc_check_chance,
+                dclocal_read_repair_chance, default_time_to_live, extensions,
+                flags, gc_grace_seconds, max_index_interval,
+                memtable_flush_period_in_ms, min_index_interval, read_repair_chance,
+                speculative_retry
+                FROM system_schema.tables
+                WHERE keyspace_name='{test_keyspace}' AND table_name='{get_name(new_tbl)}'
+            """).one()
+            assert tbl_row == new_tbl_row
+
+            tbl_scylla_row = cql.execute(f"""
+                SELECT cdc, partitioner
+                FROM system_schema.scylla_tables
+                WHERE keyspace_name='{test_keyspace}' AND table_name='{get_name(tbl)}'
+            """).one()
+            new_tbl_scylla_row = cql.execute(f"""
+                SELECT cdc, partitioner
+                FROM system_schema.scylla_tables
+                WHERE keyspace_name='{test_keyspace}' AND table_name='{get_name(new_tbl)}'
+            """).one()
```

Comment:
Should also check the content of `system_schema.tables` and `system_schema.columns` too. `system_schema.scylla_tables` only contains the scylla specific additions on top of `system_schema.tables` (which we inherited from C*).